wallet/chainntfns.go

```diff
@@ -889,6 +890,24 @@ func (w *Wallet) VoteOnOwnedTickets(ctx context.Context, winningTicketHashes []*
 				ticketVoteBits = tvb
 			}
 
+			// When not on mainnet, randomly disapprove blocks based
+			// on the disapprove percent.
+			dp := w.DisapprovePercent()
+			if dp > 0 {
+				if w.chainParams.Net == wire.MainNet {
+					log.Warnf("block disapprove percent set on mainnet")
+				} else {
+					rand.Seed(time.Now().UnixNano())
```

Comment:
Don't set the seed here, since it would prevent deterministically testing the code. Seeding the PRNG is usually done only once at startup time.

Reply:
Set in init ok?